requirements/system/server.tex

```diff
@@ -1,8 +1,55 @@
 \subsection{Central Server}
+\subsubsection{Definitions}
 \begin{enumerate}
- \item CSs must be accessible through a CLI.
- \item CSs must store data on queued/running/past jobs in a database.
- \item CS job database must not change on CS crash/restart.
- \item Users must be able to query the state of jobs and work machines from a CSs via a web interface or API.
+  \item[NCS10] A job in the context of the central server consists of: (TODO: define here or in WMC??)
```

Comment:
As for the TODO, we can put definitions in its own section, I think they apply for both WMC and SC, as well as for the UC.